sfc/visitors.html

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Welcome Ingress Visitors!
+header: Welcome Ingress Visitors!
+---
+  <p>
+    Welcome to the Visitor area for the San Francisco Resistance team.
+  </p>
+
```

Comment:
We have a link down at the bottom of the page "list of outside san francisco communities", but good idea, if they got to the visitor's page, they should be able to get back to the new players page.